tasks/config.yml

```diff
@@ -1,17 +1,38 @@
 ---
 # Configure RabbitMQ
 
-- name: rabbitmq config file
-  template: src=etc/rabbitmq/rabbitmq.config.j2 dest=/etc/rabbitmq/rabbitmq.config owner=root group=root mode=0644
```

Comment:
What happened to the config file? Forgot the re-add again? It is required for RabbitMQ instlalations/clusters...

Reply:
this actually is a severe issue, and that PR was already merged for some reason

config file is not being copied at all